_tutorials/new-component/create-a-new-component-part-1.md

```diff
@@ -0,0 +1,237 @@
+
+Although Origami components use Bower to install dependencies, developer dependencies may be installed using the [NPM](https://www.npmjs.com/) package manager, as seen in `package.json`. Rules for package management are defined in the [package management section of the specification](/spec/v1/components/#package-management).
+
+Although Origami components are authored using Bower, components are published to NPM so projects which use Origami may choose to use NPM over Bower ([but we still recommended Bower for now](/docs/tutorials/npm/)).
+
+Don't worry if this sounds a little confusing at the moment. Later in this tutorial we will install dependencies using Bower for our component to use. We will also see how Origami components are automatically published to NPM.
+
+## Start Developing
+
+Now we have a basic component to work from we can start developing!
+
+First we need to install any component dependencies by running the Origami Build Tools `install` command:
+
+<pre><code class="o-syntax-highlight--bash">obt install</code></pre>
+
+Then we can use the `develop` (`dev`) command to start working on the component. The `dev` command creates a server for us to preview our component in the browser. And whenever we make any change to the source code the component will be rebuilt, which we will be able to see by refreshing our browser:
+
+<pre><code class="o-syntax-highlight--bash">obt dev</code></pre>
+
+Opening the link output by the develop command, for example `localhost:8999`, shows us the component demos (html files) and demo assets (built js and css files) in the browser.
+
+<figure>
+	<img alt="" src="/assets/images/tutorial-new-component/example-dev-output.png" />
+	<figcaption class="o-typography-caption">
+        Running the dev command builds our demos. We'll discuss demos more shortly.
+	</figcaption>
+</figure>
+
+<figure>
+	<img alt="" src="/assets/images/tutorial-new-component/example-demos-local.png" />
+	<figcaption class="o-typography-caption">
+        Opening the link output by the develop command, `localhost:8999` in this case, shows us the component demos (html files) and demo assets (js and css files) in the browser.
+	</figcaption>
+</figure>
+
+Clicking `demo.html` to open that demo will show a blank page. In the next section we will update this demo with markup and content for our component.
+
+## Markup
+
+So Origami components can be used with any backend language or framework users of Origami components copy and paste HTML from the readme `README.md` or demos in the [Origami registry](https://registry.origami.ft.com/components/) (although templates may be provided in some cases [according to the markup section of the component specification](/spec/v1/markup/#templates)). To update the markup in our component demos see the `demos` directory.
```

Comment:
```suggestion
Origami components are not built to work with any particular framework in mind. This is because product developers may choose to use any technology stack to built their application, and it’s important that they not be forced to choose a particular one in order to use Origami components. With that in mind Origami components are not written in a templating language or a framework but in plain <abbr title="Hyper Text Markup Language">HTML</abbr> and users of Origami components copy and paste the HTML from the readme `README.md` or demos in the [Origami registry](https://registry.origami.ft.com/components/). To update the markup in our component demos see the `demos` directory.
```

I've rewritten this into _hopefully_ smaller sentences and with the aim of giving more explanation about why we don't use a framework or templating language when distributing components to users.

Reply:
looks good, there are a couple of typos in the suggestion but I can take care of those when addressing all the feedback